rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/60_add_dimensions.yml

```diff
@@ -114,8 +114,8 @@ add dimensions to no dims with dynamic_template over bulk:
                 mode: time_series
                 routing_path: [metricset]
                 time_series:
-                  start_time: 1
-                  end_time: 253402300799998
+                  start_time: 2021-04-28T00:00:00Z
+                  end_time: 2999-04-28T00:00:00Z
```

Comment:
Maybe use something closer to the range of data we have?

Reply:
done, and there left some failed tests, I will run gradle check to find all failed tests, and fix it tomorrow